src/controllers/update/update.review.beneficial.owner.individual.ts

```diff
@@ -4,26 +4,28 @@ import {
   UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE
 } from "../../config";
 import { NextFunction, Request, Response } from "express";
-import { getApplicationData, removeFromApplicationData, setApplicationData } from "../../utils/application.data";
+import { getApplicationData, mapDataObjectToFields, removeFromApplicationData, setApplicationData } from "../../utils/application.data";
 import { logger } from "../../utils/logger";
 import { BeneficialOwnerIndividualKey } from "../../model/beneficial.owner.individual.model";
 import { ApplicationDataType } from "../../model";
 import { setBeneficialOwnerData } from "../../utils/beneficial.owner.individual";
 import { v4 as uuidv4 } from "uuid";
 import { Session } from "@companieshouse/node-session-handler";
 import { saveAndContinue } from "../../utils/save.and.continue";
-import { EntityNumberKey, InputDate } from "../../model/data.types.model";
+import { AddressKeys, EntityNumberKey, InputDate } from "../../model/data.types.model";
 import { addCeasedDateToTemplateOptions } from "../../utils/update/ceased_date_util";
 import { CeasedDateKey } from "../../model/date.model";
+import { ServiceAddressKey, ServiceAddressKeys } from "../../model/address.model";
 
 export const get = (req: Request, res: Response) => {
   logger.debugRequest(req, `${req.method} ${req.route.path}`);
   const appData = getApplicationData(req.session);
   const index = req.query.index;
 
-  let dataToReview = {};
+  let dataToReview = {}, serviceAddress = {};
   if (appData?.beneficial_owners_individual){
     dataToReview = appData?.beneficial_owners_individual[Number(index)];
+    serviceAddress = (dataToReview) ? mapDataObjectToFields(dataToReview[ServiceAddressKey], ServiceAddressKeys, AddressKeys) : {};
```

Comment:
We must have an issue on clicking the change link where the residential address is not populated? Out of scope for this bug ticket, but same area.